data-types/objects/src/exercises/__tests__/1_assigning_values.test.js

```diff
@@ -0,0 +1,113 @@
+/*eslint-disable no-unused-expressions */
+`
+⬇️
+
+Welcome to Assigning Values
+
+This session is designed to introduce you to how assigning values to variables
+works in javascript. In javascript the typeof a value determines whether the
+variable is assigned by value or assigned by refererence.
+This exercise will discuss and demonstrate the implications of this difference.
+
+⬆️
+`;
+
+`📚 Exercise 1 - Assign by Value 📚
+
+In this exercise, we discuss the data types in javascript assigned by value.
+Numbers, Strings, Booleans, undefined, null, and Symbols are the different
+types of variables that are assigned by value. As a result, the variable's
+value can only be changed by reassigning the variable itself
+(The assigned value cannot be mutated).
```

Comment:
In this context, what do you mean when you say the assigned value cannot be mutated? Is the restriction from mutation due to the type of the variable (number/string/boolean), or due to the fact that it is pass by value? Could you make this relationship of these concepts more explicit?